test_pkgs/ament_lint/CMakeLists.txt

```diff
@@ -11,6 +11,6 @@ find_package(ament_cmake REQUIRED)
 if(BUILD_TESTING)
   find_package(ament_cmake_gtest REQUIRED)
   ament_add_gtest(${PROJECT_NAME} test.cpp)
-endif()
+endif()  
```

Comment:
@rhaschke I just saw this but you should configure your editor to trim trailing whitespace on save automatically